login.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+require_once 'connectToDB.php';
+require_once 'src/Models/UsersModel.php';
+
+session_start();
+function verifyLogin(): void
```

Comment:
Don't forget you need to redirect the user to index.php if they are already logged in